contribute/design/1.x/cstor-operator/doc.md

```diff
@@ -181,7 +227,45 @@ type CStorPoolInstanceSpec struct {
 }
 
 
+// CStorPoolInstanceStatus is for handling status of pool.
+type CStorPoolInstanceStatus struct {
+	
+	// Current state of CSPI with details.
+	Conditions []CStorPoolInstanceCondition 
+	//  The phase of a CStorPool is a simple, high-level summary of the pool state on the 
+	//  node.  
+	Phase    CStorPoolPhase        `json:"phase"`
+	// Capacity describes the capacity details of a cstor pool 
+	Capacity CStorPoolCapacityAttr `json:"capacity"`
+	// A human readable message indicating details about why the CSPI is in this
+  // condition.
+	Message            string      `json:"message,omitempty"`
```

Comment:
Should we need to remove it from here? There are no conditions in `CStorPoolInstanceStatus`.

Reply:
Rest of the things looks good

Reply:
Thanks. Done.